crew.tscn

```diff
@@ -424,16 +435,53 @@ libraries = {
 "": SubResource("AnimationLibrary_lynlg")
 }
 
-[node name="Area2D" type="Area2D" parent="."]
+[node name="BodyArea" type="Area2D" parent="."]
 collision_layer = 32768
 collision_mask = 32768
 metadata/_edit_lock_ = true
 
-[node name="CollisionShape2D" type="CollisionShape2D" parent="Area2D"]
+[node name="BodyCollisionShape" type="CollisionShape2D" parent="BodyArea"]
 position = Vector2(-1, -123)
 shape = SubResource("CapsuleShape2D_ynwli")
 debug_color = Color(0.678431, 0.356863, 0.894118, 0.419608)
 
+[node name="CrewAI" type="Node" parent="."]
+script = ExtResource("10_2np2b")
+
+[node name="Agent" type="Node" parent="CrewAI"]
+unique_name_in_owner = true
+script = ExtResource("11_y45sd")
+
+[node name="IdleAction" type="Node" parent="CrewAI/Agent"]
+script = ExtResource("11_kpeov")
+action_name = &"idle"
+
+[node name="DefaultConsideration" type="Node" parent="CrewAI/Agent/IdleAction"]
+script = ExtResource("12_3vs40")
+curve = SubResource("Curve_j4pgs")
+consideration_name = &"default"
+
+[node name="WorkAction" type="Node" parent="CrewAI/Agent"]
+script = ExtResource("13_2nqfg")
+action_name = &"work"
+
+[node name="IsWithinWorkingHoursConsideration" type="Node" parent="CrewAI/Agent/WorkAction" node_paths=PackedStringArray("node")]
+script = ExtResource("13_bshj5")
+node = NodePath("../../../..")
+method = &"is_within_working_hours"
+consideration_name = &"is_within_working_hours"
+
+[node name="IsAssignedConsideration" type="Node" parent="CrewAI/Agent/WorkAction" node_paths=PackedStringArray("node")]
+script = ExtResource("13_bshj5")
+node = NodePath("../../../..")
+method = &"is_assigned"
+consideration_name = &"is_assigned"
+
+[node name="DecisionTimer" type="Timer" parent="CrewAI"]
```

Comment:
Is it bad I immediately thought about the kuji timer?!

Reply:
🫠😂